docs/manual-installation.md

```diff
@@ -24,6 +24,11 @@ This will deploy the operator in the `default` namespace.
     kubectl apply -f deploy/role_binding.yaml
     ```
 
+1. Deploy the IngressClass (if running kubernetes version >= 18.0):
+    ```
+    kubectl apply -f deploy/ingress-class.yaml
```

Comment:
@lucacome 
Users can deploy multiple Ingress Controllers using the Operator, not one. Each one will need its own Ingress class. Note that the `NginxIngressController` resource has two fields - `ingressClass` and `useIngressClassOnly`.

Could you treat IngressClass resource the way it is treated in our Helm chart? -- for k8s >= 1.18, helm creates an Ingress class that matches the `ingressClass` configured in the chart. 

It also makes sense to change the docs for `ingressClass` and `useIngressClassOnly` fields similarly to how they were changed in the Helm chart. It makes sense to mention OpenShift versions as well.